common/rootfs/docker-entrypoint.sh

```diff
@@ -70,6 +70,12 @@ if bashio::config.true 'disable_tuya_default_response'; then
     export DISABLE_TUYA_DEFAULT_RESPONSE="true"
 fi
 
+TIMEZONE=$(bashio::config 'timezone')
+
+if [[ -n "$SOCAT_MASTER" ]]; then
```

Comment:
Why only set the `TZ` if `SOCAT_MASTER` is not empty?

Reply:
Sorry, copy-paste fail. I meant to check whether `TIMEZONE` is non-empty.